lib/ansible/modules/windows/win_lineinfile.ps1

```diff
@@ -311,7 +311,7 @@ $diff_support = Get-AnsibleParam -obj $params -name "_ansible_diff" -type "bool"
 
 # Initialize defaults for input parameters.
 $path = Get-AnsibleParam -obj $params -name "path" -type "path" -failifempty $true -aliases "dest","destfile","name";
-$regexp = Get-AnsibleParam -obj $params -name "regexp" -type "str";
+$regexp = Get-AnsibleParam -obj $params -name "regexp" -type "str" -aliases "regex";
```

Comment:
It looks like the decision was to standardise on `regex` as being the option name and have `regexp` as the alias for that option. While this is the code the actual docs file should reflect that. It would be nice if the code was also the same to make it more maintainable in the future.

Reply:
done